test/connectivity/scripts/connectivityTest.sh

```diff
@@ -346,7 +352,8 @@ function run_connectivity_test() {
         -t "$ARTIFACT_IMAGE_BUILD_NUMBER-linux-$image_architecture_label" \
         --leave-running=All \
         -l "$deployment_working_file" \
-        --runtime-log-level "Debug" \
+        --runtime-log-level "Info" \
+        --optimize_for_performance=$optimize_for_performance \
```

Comment:
Did you ever figure out why the edgehub optimize for performance was being set to false when, according to the quickstart implementation, this flag should not do anything?

(Context: this flag is ignored if there is a deployment file.)

Reply:
In the Dockerfiles for ARM32v7 and ARM64v8, they already set the `ENV OptimizeForPerformance false` by default. I forgot to remove the line.